code/controllers/subsystems/overlays.dm

```diff
@@ -209,7 +209,7 @@ SUBSYSTEM_DEF(overlays)
 		if(cut_old)
 			our_overlays = cached_other.Copy()
 		else
-			our_overlays |= cached_other
+			LAZYDISTINCTADD(our_overlays, cached_other)
```

Comment:
this change doesn't do anything, does it? `null |= list(foo)` == `list(foo)`. i think `null |= null` might be an issue but we know `cached_other` is non-null here.

Reply:
uh. it is entirely impossible for cached_other to ever be `0`, what was the runtime specifically? i mean maybe it was `"0"` in which case it'd be fine. but actually no, cached_other should always be a list or null